travis/licenses_golden/licenses_third_party

```diff
@@ -21148,14 +10334,14 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ====================================================================================================
 LIBRARY: icu
-ORIGIN: ../../../third_party/icu/source/i18n/decNumber.c + ../../../third_party/icu/LICENSE
+ORIGIN: ../../../third_party/icu/source/i18n/decNumber.cpp + ../../../third_party/icu/LICENSE
 TYPE: LicenseType.bsd
-FILE: ../../../third_party/icu/source/i18n/decNumber.c
+FILE: ../../../third_party/icu/source/i18n/decNumber.cpp
 ----------------------------------------------------------------------------------------------------
 Copyright (c) IBM Corporation, 2000-2014.  All rights reserved.
 
-                   Institute of Information Science, Academia Sinica.
-All rights reserved.
+                   Institute of Information Science, Academia
+  *                    Sinica. All rights reserved.
```

Comment:
Any idea why this extra asterisk appeared?

Reply:
That line in icu/LICENSE starts with a `#`, followed by extra whitespace and a `*` that forms part of a C-style comment.  I guess the indent regex captures the `*` because of the additional whitespace?